src/components/DropDown.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from "classnames";
+import React from "react";
+import { Person } from "../types/Person";
+
+type Props = {
+  items: Person[];
+  onSelect: (person: Person) => void;
+  setQuery: (name: string) => void;
+}
+
+export const DropDown: React.FC<Props> = React.memo(({
+  items,
+  onSelect,
+  setQuery,
+}) => {
+
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {items.map(person => (
+          <div
+            key={person.slug}
+            className="dropdown-item"
+            onMouseDown={() => {
+              onSelect(person);
+              setQuery(person.name)
+            }}
```

Comment:
Pls move the function inside `onMouseDown` handler to a separate const and use as a callback here